packages/logger-plugin/src/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { ProdoPlugin } from "@prodo/core";
+
+export interface LoggerConfig {
+  logger?: boolean;
+}
+
+const init = (config: LoggerConfig) => {
+  if (config.logger) {
+    // tslint:disable-next-line
+    console.log("@prodo/logger is on");
```

Comment:
We probably want to disable this in `NODE_ENV === "production"` mode

Reply:
Users will generally want to disable the entire plugin in production, but I would probably leave that to them for now, rather than hardcoding it in the plugin itself.

Reply:
Right, thats a good point